CHANGES.md

```diff
@@ -67,6 +67,8 @@
 
 * [Enrichment Transform](https://s.apache.org/enrichment-transform) along with GCP BigTable handler added to Python SDK ([#30001](https://github.com/apache/beam/pull/30001)).
 * Allow writing clustered and not time partitioned BigQuery tables (Java) ([#30094](https://github.com/apache/beam/pull/30094)).
+* Merged sdks/java/fn-execution and runners/core-construction-java into the main SDK. These artifacts were never meant for users, but noting
```

Comment:
This is unfortunately an unpleasant breaking change. We don't have much context why `core-construction-java` was never meant for users since we use some of the classes. This change makes it impossible for us to rollout to our users.

Reply:
We will appreciate at least this was mentioned in the release notes.